src/shared/components/comment/comment-node.tsx

```diff
@@ -1209,7 +1211,7 @@ export class CommentNode extends Component<CommentNodeProps, CommentNodeState> {
 
     const title = this.props.showContext
       ? I18NextService.i18n.t("show_context")
-      : I18NextService.i18n.t("link");
+      : I18NextService.i18n.t("link parent");
```

Comment:
I don't understand this one. Almost all the cases of CommentNode / CommentNodes have the show context flag, they only don't for things like the comment report. In which case the link is fine.